app/views/govuk_publishing_components/components/docs/document_list.yml

```diff
@@ -1,16 +1,16 @@
 name: Document list
 description: An ordered list of documents including a document type, when updated and a link.
 body: |
-  Outputs a list to documents, based on an array of document data. This may include:
+  Outputs a list to documents, based on an array of document data. A "document" in this context is considered any page on the web, so the document list can be considered a list of references (links, titles etc) to web pages.
+  
+  This component may include:
```

Comment:
I think this would be better written as `This may include` as it's referring to the list of things, rather than the component?

Reply:
I read this list as elements that can be part of the component. We've just been talking about something else (what a document is) so I thought it was helpful to qualify what this list is, otherwise people will think this list expands on what a document is. Have I gone astray?

Reply:
That's a good point, hadn't thought of that. How about `The component can display:`?